ecgtools/core.py

```diff
@@ -288,12 +297,16 @@ def update(self, catalog_file: str, path_column: str) -> 'Builder':
         self.old_df = pd.read_csv(catalog_file)
         filelist_from_prev_cat = self.old_df[path_column].tolist()
         filelist = self._get_filelist_from_dirs()
+
+        if local_attrs is None:
+            local_attrs = {}
+
```

Comment:
Same as above.

Reply:
@kmpaul , I made the same change as in the other comment.